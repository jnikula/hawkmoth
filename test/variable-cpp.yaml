domain: cpp
directive-arguments:
  - variable.c
directive-options:
  clang:
    - -xc++
