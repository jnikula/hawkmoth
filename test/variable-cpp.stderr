ERROR: variable.c:34: default initialization of an object of const type 'const char *(*const)(const char *)'
ERROR: variable.c:39: default initialization of an object of const type 'const char *(*const volatile)(const char *)'
ERROR: variable.c:44: default initialization of an object of const type 'const char *(*const *volatile *const[12][3])(const char *)'
